test/src/watcher/watcher.test.ts

```diff
@@ -83,6 +84,8 @@ describe("Watcher", () => {
     when(mockedReorgDetector.addReorgHeightListener(anyNumber(), anything())).thenReturn();
     const reorgDetectorInstance = instance(mockedReorgDetector);
 
+    const mockedBlockProcessor = mock(BlockProcessor);
```

Comment:
Seems like a left over from refactoring, not needed any more

Reply:
Removed.